.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
VScode setup for remote debugging. just click on `CLRT Docker Django` option from VSCode editor Debug option from left Nav. Add a breakpoint where you want to debug code

![Screenshot 2024-06-19 at 3 12 58 PM](https://github.com/tl-its-umich-edu/canvas-lti-redirect-tool/assets/8579775/6a7359af-116c-46b9-97a4-b735999a69a8)